tests/adcc/CMakeLists.txt

```diff
@@ -18,3 +18,6 @@ add_subdirectory(h2o-adc2-any)
 add_subdirectory(cn-adc1)
 add_subdirectory(cn-adc2)
 add_subdirectory(hf-adc2-spin-flip)
+
+# Specific properties
+add_subdirectory(methyloxirane_rotation)
```

Comment:
I think you're mixing underscore and dash in the dir/test name. go for dash, please.